src/Tasks/Microsoft.NETFramework.CurrentVersion.props

```diff
@@ -85,6 +85,7 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
   <PropertyGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework' and '$(TargetingClr2Framework)' != 'true' and '$(TargetFrameworkVersion)' != 'v4.0' and ('$(OutputType)' == 'exe' or '$(OutputType)' == 'winexe' or '$(OutputType)' == 'appcontainerexe' or '$(OutputType)' == '')">
     <Prefer32Bit Condition="'$(Prefer32Bit)' == ''">true</Prefer32Bit>
+    <PreferNativeArm64 Condition="'$(PreferNativeArm64)' == ''">false</PreferNativeArm64>
```

Comment:
Why set this explicitly, instead of letting it be default/empty?

Reply:
I did it according to https://github.com/dotnet/msbuild/issues/10060#:~:text=to%20false.%20PreferNativeArm64-,is%20a%20false%2Dby%2Ddefault,-opt%2Din%20switch

Have I misunderstood it? :)

Reply:
There are two ways to make something false-by-default in MSBuild:

1. Set the property in a .targets file (late in eval, after the project body) to a default value of `false` if it's not set (like you're doing here)
2. Treat "unspecified"/"empty string" expansions of that property as `false`.

I tend to prefer the latter, but it's a judgement call; I was asking if you have a specific reason to prefer the former.

Reply:
Thank you for explanation, I will clean it up